lib/metadata.go

```diff
@@ -97,6 +97,62 @@ func (c *MasterMetadataClient) GenerateEpoch(ctx context.Context) (Epoch, error)
 	return resp.Header.Revision, nil
 }
 
+type UserMetadataClient struct {
```

Comment:
the meta client PR has been merged, we can use it now.

Reply:
on it